test/e2e/e2e_topologyspreadconstraint_test.go

```diff
@@ -194,9 +196,6 @@ func TestTopologySpreadConstraint(t *testing.T) {
 				return
 			}
 
-			// Ensure recently evicted Pod are rescheduled and running before asserting for a balanced topology spread
-			test.WaitForDeploymentPodsRunning(ctx, t, clientSet, deployment)
```

Comment:
This is needed for synchronization as the comment describes.

Reply:
We are currently running the descheduler inside a container, so we just need to continuously wait for the results using `wait.PollUntilContextTimeout`. We don’t actually know when the descheduler evicts the pods.
https://github.com/kubernetes-sigs/descheduler/blob/5464f377a10f217fe4c35a756fcdf43ca680a7a2/test/e2e/e2e_topologyspreadconstraint_test.go#L258